woocommerce/woocommerce
# [Launch Your Store] Add e2e tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/46568

Add e2e tests around Launch Your Store. There are 4 scenarios tested to make sure the front end is displaying properly depending on Coming soon mode and logged in status.

* Entire site logged in and logged out
* Store only logged in and logged out



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Read through the added tests and code changes to make sure the logic is ok
2. See tests pass on CI



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Add e2e test to unreleased feature

</details>
